lib/spect.ex

```diff
@@ -153,8 +163,18 @@ defmodule Spect do
       )
   end
 
+  defp to_type!(data, module, :module, _args) do
```

Comment:
This function isn't covered by the tests. Run `MIX_ENV=test mix coveralls.html` for code coverage.